internal/common/utils.go

```diff
@@ -75,7 +91,10 @@ func GetConfigMaps(ctx context.Context, c client.Client, configMaps []v1alpha1.C
 
 // PathOutsideChroot returns filepath with host fs
 func PathOutsideChroot(filename string) string {
-	return filepath.Join(Host, filename)
+	if hostDirExists {
```

Comment:
It seemed slightly more efficient to just check the bool directly in this case, rather than having the added function call. The accessor function was added in case there was a need to check for /host outside this module

Reply:
Removed the caching